src/main/java/com/commercetools/sync/commons/enums/Error.java

```diff
@@ -4,15 +4,26 @@
 import javax.annotation.Nonnull;
 
 /**
- * Represents all error messages across the sync update action build modules.
+ * Represents all error messages across the sync modules.
  */
 public enum Error {
```

Comment:
What is the gain from having all update messages in one place as public enum anyway? They are used always as ENUM.getDescritpion() inside format for calllback message. Why they're not kept as `private final static String` in context they belong to? Am I missing something?